src/main/resources/org/jenkinsci/plugins/autocompleteparameter/providers/RemoteDataProvider/help-prefetch.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    If enabled, prefetch results once on page load, else invokes on every text change.
```

Comment:
If enabled, prefetch results once on page load. Otherwise, will query the remote endpoint on every text change.